jcl/src/java.base/share/classes/java/lang/System.java

```diff
@@ -147,8 +147,13 @@
 		sysPropOSVersion = getSysPropBeforePropertiesInitialized(sysPropID_OSVersion);
 		/*[ENDIF] JAVA_SPEC_VERSION >= 18 */
 		platformEncoding = getSysPropBeforePropertiesInitialized(sysPropID_PlatformEncoding);;
-		String definedFileEncoding = getSysPropBeforePropertiesInitialized(sysPropID_FileEncoding);
 		String definedOSEncoding = getSysPropBeforePropertiesInitialized(sysPropID_OSEncoding);
+		/*[IF JAVA_SPEC_VERSION >= 18]
+		-Dfile.encoding=COMPAT is mapped to platform encoding within getSysPropBeforePropertiesInitialized.
+		During early VM boot stage, JITHelpers hasn't been initialized yet, COMPACT_STRINGS related APIs
+		might not work properly.
```

Comment:
I'd like to see these lines prefixed with `//` to make it clearer that they are comments.

Reply:
Hmm ya, good point. I don't think the code is going to compile after going through the preprocessor.

Reply:
The text is part of a larger `/*`-style comment, but that wasn't immediately clear to me, hence my suggestion.

Reply:
Fixed.